data/mods/More_Survival_Tools/ammo.json

```diff
@@ -70,5 +70,51 @@
         "dispersion" : 0,
         "recoil" : 0,
         "count" : 200
+    },
+    {   "type" : "AMMO",
+        "id" : "pebble_clay",
+        "price" : 150,
+        "name" : "clay pellet",
+        "symbol" : "=",
+        "color" : "brown",
+        "description" : "A handful of round projectiles made of clay, useful for slings or slingshots.",
+        "material" : "clay",
+        "volume" : 1,
+        "weight" : 3,
+        "bashing" : 1,
+        "cutting" : 0,
+        "to_hit" : 0,
+        "ammo_type" : "pebble",
+        "casing" : "NULL",
+        "damage" : 2,
+        "pierce" : 0,
+        "range" : 20,
+        "dispersion" : 13,
```

Comment:
You're using the old values of dispersion here (before it all got multiplied by 15). Is the rest of the sling-related stuff still in old dispersion?